Osiris/Hacks/Visuals.cpp

```diff
@@ -393,3 +399,25 @@ void Visuals::skybox() noexcept
         memory->loadSky(sv_skyname->string);
     }
 }
+
+std::wstring towstring(std::string str)
+{
+    return strconverter.from_bytes(str);
+}
+
+void Visuals::showVelocity() noexcept
+{
+    if (!config->visuals.showvelocity || !localPlayer)
+        return;
+
+    double velocity = localPlayer->velocity().length2D();
+    std::stringstream velocitystream;
+    velocitystream << std::fixed << std::setprecision(0) << velocity;
+    std::wstring velocitywstr = towstring(velocitystream.str());
```

Comment:
can be done better
`std::wstring velocitywstr{ L"Velocity: " + std::to_wstring(static_cast<float>(velocity)) };`
can change from float to int, if you want to remove decimals

Reply:
done thanks